test/spec/xml/roundtrip.js

```diff
@@ -536,14 +536,10 @@ describe('bpmn-moddle - roundtrip', function() {
         rootElement
       } = await fromFile('test/fixtures/bpmn/ad-hoc.bpmn');
 
-      // cancelRemainingInstances should not have a default value
-      expect(rootElement.get('cancelRemainingInstances')).not.to.exist;
```

Comment:
@philippfromme The correct way to check if the value is present on the XML is via string comparison. If a default value is accessed via `moddleElement#get`, it will always exist.